docs/sources/mimir/configure/configure-ingester-circuit-breakers.md

```diff
@@ -0,0 +1,83 @@
+- a read request that acquired a read requests circuit breaker permit meets the read requests circuit breaker failure condition if its duration is longer than the configurable read request maximum duration.
+
+## Grafana Mimir Ingester circuit breakers configuration
+
+Grafana Mimir Ingester circuit breakers can be configured by using the following configuration options.
+
+- `-ingester.push-circuit-breaker.enabled`: Enable circuit breaking when making push requests to ingesters.
+
+- `-ingester.read-circuit-breaker.enabled`: Enable circuit breaking when making read requests to ingesters.
+
+- `-ingester.push-circuit-breaker.thresholding-period`: Moving window of time that the percentage of failed requests is computed over for push circuit breakers.
+
+- `-ingester.read-circuit-breaker.thresholding-period`: Moving window of time that the percentage of failed requests is computed over for read circuit breakers.
+
+- `-ingester.push-circuit-breaker.failure-execution-threshold`: How many push requests must have been executed within the thresholding period for a push circuit breaker to be eligible to open for the rate of failures?
+
+- `-ingester.read-circuit-breaker.failure-execution-threshold`: How many read requests must have been executed within the thresholding period for a read circuit breaker to be eligible to open for the rate of failures?
+
+- `-ingester.push-circuit-breaker.failure-threshold-percentage`: Max percentage of push requests that can fail over period before a push circuit breaker opens.
+
+- `-ingester.read-circuit-breaker.failure-threshold-percentage`: Max percentage of read requests that can fail over period before a read circuit breaker opens.
+
+- `-ingester.push-circuit-breaker.cooldown-period`: How long a push circuit breaker will stay in the open state before allowing additional push requests?
+
+- `-ingester.read-circuit-breaker.cooldown-period`: How long a read circuit breaker will stay in the open state before allowing additional read requests?
+
+- `-ingester.push-circuit-breaker.initial-delay`: How long a push circuit breaker should wait between an activation request and becoming effectively active? During that time both failures and successes are not counted.
+
+- `-ingester.read-circuit-breaker.initial-delay`: How long a read circuit breaker should wait between an activation request and becoming effectively active? During that time both failures and successes are not counted.
+
+- `-ingester.push-circuit-breaker.request-timeout`: The maximum duration of a push request before it triggers a timeout. This configuration is used for push circuit breakers only, and the corresponding timeouts are not reported as errors.
+
+- `-ingester.read-circuit-breaker.request-timeout`: The maximum duration of a read request before it triggers a timeout. This configuration is used for read circuit breakers only, and the corresponding timeouts are not reported as errors.
+
+## Grafana Mimir Ingester circuit breakers metrics
```

Comment:
Please make it clear that metrics aren't part of any API guarantee and can be changed at any time.